src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -292,13 +292,12 @@ public void close(final Collection<TopicPartition> partitions) {
    */
   @Override
   public void put(final Collection<SinkRecord> records) {
+    final long startTime = System.currentTimeMillis();
     final long recordSize = records.size();
     if (enableRebalancing && recordSize > 0) {
       processRebalancingTest();
     }
 
-    final long startTime = System.currentTimeMillis();
-
     getSink().insert(records);
 
     logWarningForPutAndPrecommit(
```

Comment:
We probably want INFO level log on this

Reply:
Changed to info - should we add the log level as a configurable param? We've changed the log level for these logs many times...

[1/18: customer issue info -> debug ](https://github.com/snowflakedb/snowflake-kafka-connector/commit/cdfee1b38a7956af401f26c54f26efeff69c3229)
[6/8: debug -> info](https://github.com/snowflakedb/snowflake-kafka-connector/commit/a984dcff3440106b407487678b5167bbaeb426a1)
[6/27: info -> debug](https://github.com/snowflakedb/snowflake-kafka-connector/commit/cfcf882eae846841e85569d1888b4bb8c6e410dd)

Reply:
Customers can change it to debug if they like.